firefox_puppeteer/api/utils.py

```diff
@@ -0,0 +1,76 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette.errors import TimeoutException
+
+from ..base import BaseLib
+
+
+class Utils(BaseLib):
+    """Low-level access to utility actions."""
+
+    def sanitize(self, data_type):
+        """Sanitize user data, including cache, cookies, offlineApps, history, formdata,
+        downloads, passwords, sessions, siteSettings.
+
+        Usage:
+        sanitize():  Clears all user data.
+        sanitize({ "sessions": True }): Clears only session user data.
+
+        more: https://dxr.mozilla.org/mozilla-central/source/browser/base/content/sanitize.js
+
+        :param data_type: optional, Information specifying data to be sanitized
+        """
+
+        try:
+            self.marionette.execute_async_script("""
```

Comment:
> I just mean that in the current arrangement, if the promise is rejected
> the test will not fail. I'm not saying it's critical to the test, but
> you can fail the test in this case by just checking the return of
> execute_async_script and throwing an exception if it's false.

That's exactly what I was mentioning in my last review. We have to raise
an exception here! Otherwise tests which require a clean environment
will fail with unknown reasons. So please let an exception be raised here.